db/migrate/20220105024605_add_trigger_for_polymorphic_bookmark_columns_to_sync_data.rb

```diff
@@ -28,21 +28,29 @@ def up
     SQL
 
     # sync data that already exists in the table
-    DB.exec(<<~SQL)
-      UPDATE bookmarks
-      SET bookmarkable_id = post_id, bookmarkable_type = 'Post'
-      WHERE NOT bookmarks.for_topic
-    SQL
-    DB.exec(<<~SQL)
-      UPDATE bookmarks
-      SET bookmarkable_id = posts.topic_id, bookmarkable_type = 'Topic'
-      FROM posts
-      WHERE bookmarks.for_topic AND posts.id = bookmarks.post_id
-    SQL
+    #
+    # Note from Martin:
+    #
+    # We cannot remove this migration but we also don't want to do this
```

Comment:
Sorry quick clarification, what is the reason we can't remove this entire migration completely?

Reply:
Do we often delete migrations? I think that is a bad idea generally.

Reply:
I did a quick search internally and we do have a directive to not remove migrations. Ignore me here :grin:

Reply:
I thought so, I'm not exactly sure what it can mess up but I know it's one of those "messing with Rails magic" kind of things :) At least the backfilling which was the biggest part of this won't happen now.